autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -33,6 +34,44 @@ const CourseSelectColumn: React.FC = () => {
     return (): void => { dispatch(clearCourseCards()); };
   }, [term, dispatch]);
 
+  // When courseCards is changed, save courses (and throttle API calls)
```

Comment:
I'd change this to something like `When courseCards is changed, set off a timer that will save the courses in 15 seconds. Previous timer is invalidated if this gets called again (when course cards are further updated)` (but more succinctly lol)